packages/plugin-hardhat/contracts/GreeterProxiable40FallbackString.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity >= 0.6.0 <0.8.0;
+
+import { Proxiable } from "./utils/Proxiable.sol";
+
+contract GreeterProxiable40FallbackString is Proxiable {
+
+    string greeting;
+
+    function initialize(string memory _greeting) public {
+        greeting = _greeting;
+    }
+
+    function greet() public view returns (string memory) {
+        return greeting;
+    }
+
+    function setGreeting(string memory _greeting) public {
+        greeting = _greeting;
+    }
+
+    fallback(bytes calldata) external returns (bytes memory) {
+        return abi.encode(greeting);
```

Comment:
Does it not work with a direct cast 
```suggestion
        return bytes(greeting);
```
?

Using `abi.encode` does an extra encapsulation. Not sure that is needed.

Reply:
It actually was needed to force the return type to look like a string (added a test in https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/926/commits/521f1acde17da7f075dcf0819518c6cb30f5e886)